wiki/Ranking_Criteria/en.md

```diff
@@ -133,6 +133,7 @@ Custom naming must follow a common theme or pattern related to the song and must
 
 ### Guidelines
 
+-   **The highest difficulty of a beatmapset should correspond to the general feel of the song.** Easy/Normal difficulties can be used as lone difficulties of a beatmapset if their rhythms are not oversimplified. A Hard difficulty or beyond should be included otherwise.
```

Comment:
can be used as lone difficulties of a beatmapset ->
can be used as a lone difficulty of a beatmapset
the thing referred to is singular

Reply:
done